x-pack/docs/en/security/securing-communications/tls-transport.asciidoc

```diff
@@ -30,9 +30,12 @@ See <<ssl-tls-settings, `xpack.ssl.verification_mode`>> for a description of the
 <2> If you created a separate certificate for each node, then you might need to
 customize this path on each node. If the filename matches the node name, you can
 use the `certs/${node.name}.p12` format, for example.
-<3> The `elasticsearch-certutil` output includes the CA certificate inside the
-PKCS#12 keystore, therefore the keystore can also be used as the truststore.
-This name should match the `keystore.path` value.
+<3> The `elasticsearch-certutil` outputs a PKCS#12 keystore which includes the
+CA certificate as a trusted certificate entry. This allows for the keystore to
+also be used as a truststore. In this case, the path value should match
+the `keystore.path` value.
+Note, however, that this is not the general rule. There are keystores that cannot be
+used as trustores, only <<pkcs12-trustore-note, specifically crafted ones can>>.
```

Comment:
Not sure if the 'pkcs12-trustore-note' link is working correctly here. @lcawl can you please check if this is the proper way to link?

Reply:
@albertzaharovits Things are a bit messy at the moment with OSS and X-Pack docs being split across 2 "books".
The `pkcs12-trustore-note` reference is in the ES docs (see https://www.elastic.co/guide/en/elasticsearch/reference/6.2/security-settings.html), but this page is in the X-Pack docs (see https://www.elastic.co/guide/en/elasticsearch/reference/6.2/configuring-tls.html#tls-transport).
So you need to do `{ref}/security-settings.html#pkcs12-trustore-note`

Can you also fix the spelling on `truststore`?

Reply:
That's right, with the title in []: {ref}/security-settings.html#pkcs12-trustore-note[specially crafted ones] ...

Reply:
Thank you both for the explanation!